### ISSUE TITLE
Feature Request: Embed alveusgg chat on the right side of the pages

### ISSUE THREAD

**Issue by Taizunz** [dansza1/ptz-alv#2]
Title says most of it.

I'd love to be able to see the alveusgg at a glance when I'm using the web app.
A mockup of what I'm envisioning:
![image](https://github.com/dansza1/ptz-alv/assets/15648000/b287577d-90a8-4a75-a0a0-f69cedf417dd)

**Comment by dansza1** [comment]
I dont think we have the screen real estate to fit the chat on the right like you have in your mock up? generally bad practice to force a horizontal scroll to see all elements and also makes viewing on mobile tricky.
